examples/dmabuf-capture.c

```diff
@@ -0,0 +1,743 @@
+#define _XOPEN_SOURCE 700
```

Comment:
Given the recent comments having some usage / trouble shoot  instructions here would IMHO be good